plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/IOpenShiftConnection.java

```diff
@@ -22,7 +24,7 @@
  * @author jeff.cantrill
  *
  */
-public interface IOpenShiftConnection {
+public interface IOpenShiftConnection extends IConnection {
```

Comment:
I see the convienience argument, but if there are different kinds of connection, shouldn't we remove the not-common API from the common superinterface? In other news, interfaces should tell the intention (the TRUTH!) of a structure, not a convenient lie. 
But honestly it was just a PITA to cast ;-)